### PR TITLE
Adds French Translation, as requested in Issue #12

### PR DESCRIPTION
**Pull request by ashutoshsaboo**
This PR adds French, as requested in  Issue #12 .
Please review this @Ceilican .

### CODE REVIEW

**Comment by ashutoshsaboo** [comment]
@Ceilican : This PR has passed no conflicts, so is this good to get merged now? Thanks! :smile:

File: Description.md [hunk 1]
```diff
@@ -49,9 +49,9 @@ ToDo
 French (fr)
 -----------
 
-Translated by: 
+Translated by: Ashutosh Saboo | ashutoshsaboo | https://github.com/ashutoshsaboo
```
Comment (Ceilican) [inline Description.md]: Look what was done for english:

```
(Short: Estimates the CO2 emissions for driving on routes suggested by Google Maps!)

Whenever you ask Google Maps for driving directions, this browser extension automatically estimates the total CO2 emission caused by driving on the suggested routes. The estimation is shown next to the route's total distance.

To improve the accuracy of the estimations, the options allow you to configure several parameters related to the fuel comsumption and environment-friendliness of your car.

If you have comments or questions, please send them to MapsCarbonFootprint@gmail.com.

If you like this extension, please give it a good rating, like it on facebook (http://www.facebook.com/pages/Carbon-Footprint-for-Google-Maps/217487891623173) and +1 it on Google+ (https://plus.google.com/b/112973863308036008385/112973863308036008385/posts).

Spread this extension and help us raise environmental awareness! Let's preserve the environment, stop global warming and save our planet!
```

A translation of this text to french should be added here. 

Do not use an automatic translator. The translation should be done by a native speaker of French.

**Comment by Ceilican** [comment]
Thanks, @ashutoshsaboo ! Have a look on my line comment. After you have fixed that, I will merge it.

**Comment by ashutoshsaboo** [comment]
@Ceilican Hi. Thanks for your input. 

Yes I myself speak French. I have been learning it now since 5-5.5 years. I have translated the text myself, and pushed the same as a commit. 

The above translation (which I pushed) is not the same as on Google Translate. Same can be verified in the screenshot attached below-:

selection_013

So, Can this PR get merged now? @Ceilican 

Thanks. :smile:

**Comment by Ceilican** [comment]
Yes! It is perfect now, @ashutoshsaboo ! Thank you!

**Pull request merged**